phpcs.xml.dist

```diff
@@ -66,6 +66,8 @@
 	<rule ref="Generic.Commenting.DocComment">
 		<!-- Having a @see or @internal tag before the @param tags is fine. -->
 		<exclude name="Generic.Commenting.DocComment.ParamNotFirst"/>
+		<!-- We prefer seeing nicely looking docblock comments -->
+		<exclude name="Generic.Commenting.DocComment.TagValueIndent"/>
```

Comment:
1. This should be a separate PR.
2. What are you trying to solve here ? Could you give a code example ?

Reply:
Will make a separate PR of it.
See https://travis-ci.org/wimg/PHPCompatibility/jobs/310161506